### PR TITLE
more docs about species QC

### PR DESCRIPTION
**Pull request by petrelharp**
I thought we could use some more words about how to do all this, and wrote them down here.

### CODE REVIEW

File: docs/development.rst [hunk 1]
```diff
@@ -831,13 +831,27 @@ genomes housed on Ensembl can be found `here <https://metazoa.ensembl.org/specie
...
+* `species.py`
+* `__init__.py`
+
+as well as stubbing out tests in `tests/test_{species_id}.py`.
+
+The `genome_data.py` file contains the physical map of the genome; the
+`maintenance` utility sucks down a whole lot of useful information for free.
+This file essentially puts together a data dictionary which has slots for the
+assembly accession number, the assembly name, and a dict representing the
+chromosome names and their associated lengths. If synonyns are defined (e.g.,
```
Comment (mufernando) [inline docs/development.rst:844]: synonyms

**Review by mufernando — APPROVED** [review]
the additions seem helpful to me! I just commented a minor typo.

**Comment by petrelharp** [comment]
Whoops - wrong window... =)

File: docs/development.rst [hunk 2]
```diff
@@ -868,38 +882,82 @@ Next, the species.py file will need to be edited with the species-specific infor
...
 using the 'Species QC issue template'. One or more volunteers will check items off
 the checklist, until all items have been completed satisfactorily. The QC issue,
-or the pull request, may be used for review discussion. The new species will be
-merged once the checklist is completed.
+or the pull request, may be used for review discussion.
+
+To begin reviewing (i.e., QC'ing) a species, you should state your intention on the
+QC issue (so we don't duplicate effort) and start a pull request to fill out the code.
+During the process of QC, other developers fill in the tests that are disabled
+in the `tests/test_{species_name}.py` file. For instance, a test for an
```
Comment (chriscrsmith) [inline docs/development.rst:926]: Maybe mention here that the person doing the QC should be looking up the parameters in the provided citations, without looking at the values provided by the original submitter (I think that's how it works)?

**Review by chriscrsmith — COMMENTED** [review]
Looks good to me! I offered one small suggestion, but consider also I haven't done a GC yet

**Pull request merged**